docs/03-extensions/stripe/how-to/customize-data.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Customize data sent to Stripe
+description: "This guides explains how to Customize data sent to Stripe"
+sidebar_position: 1
+---
+
+<p>{frontMatter.description}</p>
+
+:::caution WIP
+
+<span>
+  This advanced pattern must be documented with further details. While we are
+  working on it, please <ContactLink /> if you need further assistance.
+</span>
+
+:::
+
+The Stripe payment module is extensible. It leverages Front-Commerce's "data
+transform" pattern to allow developers to customize payloads sent to Stripe for
+Customer and Cart content.
+
+Both the `Customer` and `PaymentIntent` Stripe objects can be customized at
+application level. It allows to add additional metadata depending on your own
+logic. For this, you can use the `registerCustomerDataTransform` and
+`registerPaymentIntentDataTransform` methods of the Stripe loader to add your
+custom transformers.
+
+See the tests for an example (while a detailed documentation is being written):
+
+- https://gitlab.blackswift.cloud/front-commerce/front-commerce/blob/3c550dfa0142dde7da3761011379118612841de7/src/server/modules/payment-stripe/__tests__/loader.js#L77
+- https://gitlab.blackswift.cloud/front-commerce/front-commerce/blob/3c550dfa0142dde7da3761011379118612841de7/src/server/modules/payment-stripe/__tests__/loader.js#L147
```

Comment:
**issue:** Those links point to 4yrs old code !

**suggestion:** Update them to point to what was actually made in v3.

**suggestion(alt):** Write the real documentation about this feature instead of the stub from v2's doc.

Reply:
updated